packages/rest-crud/README.md

```diff
@@ -41,6 +41,67 @@ npm install --save @loopback/rest-crud
    app.controller(ProductController);
    ```
 
+## defineCrudRepositoryClass
+
+A helper method `defineCrudRepositoryClass` is exported by this module, which
+can be used for creating named repositories (based on the Model).
+
+Usage example:
+
+```ts
+const db = new juggler.DataSource({connector: 'memory'});
+const ProductRepository = defineCrudRepositoryClass(Product);
+const repo = new ProductRepository(db);
+```
+
+## Integrated example
+
+This is an example of an app using `defineCrudRestController` and
+`defineCrudRepositoryClass`.
+
+```ts
+class Product extends Entity {
+  static definition = new ModelDefinition('Product')
+    .addProperty('id', {type: 'number', id: true})
+    .addProperty('name', {type: 'string'});
+
+  @property({id: true})
+  id: number;
+
+  @property()
+  name: string;
+}
+
+export class TryApplication extends BootMixin(
+  ServiceMixin(RepositoryMixin(RestApplication)),
+) {
+  constructor(options: ApplicationConfig = {}) {
+    ...
+  }
+
+  async boot():Promise<void> {
+    await super.boot();
+
+    const db = new juggler.DataSource({connector: 'memory'});
+    const ProductRepository = defineCrudRepositoryClass(Product);
+    const repo = new ProductRepository(db);
+
+    inject('datasources.db')(ProductRepository, undefined, 0);
+    this.repository(ProductRepository);
+    this.bind('repositories.ProductRepository').to(repo);
```

Comment:
Please remove this line, the binding is already created by `this.repository(ProductRepository)`.

To avoid magic strings, you may want to get the binding key from the binding created by `this.repository` when setting up dependency injection for the controller.

```ts
const repoBinding = this.repository(ProductRepository);

// ...

inject(repoBinding.key)(ProductController, undefined, 0);

Reply:
@bajtos the app is complaining `The key 'datasources.db' is not bound to any value in context application`.

`repoBinding.key` is good suggestion.